### DISCUSSION TITLE
gRPC side by side with .net REST API in the same application

### DISCUSSION THREAD

**Discussion by kdehia** [dotnet/AspNetCore.Docs#25008] [Q&A] [answered]
Hi @JamesNK 
Thanks for all the great updates you and the team are making !

I have a question, are there any samples where it shows gRPC & REST API in the same application ? 

Regards,
Khaled

**Comment by JamesNK** [comment, marked as answer]
No, but the gRPC project template creates a mix of gRPC and REST.

The default template adds a gRPC service, and a minimal API endpoint with route "/" to tell people that a gRPC client.

**Reply by kdehia** [reply]
My bad, 
For some reasons I added `UseEndpoints ` which I shouldn't, when I used `app.MapGrpcService` all work great
My bad & thanks again ! 

Regards,
Khaled

**Reply by Cher172** [reply]
What?
